infrastructure/kube/keep-prd/bitcoin/electrumx/electrumx-volumesnapshot.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: snapshot.storage.k8s.io/v1
+kind: VolumeSnapshot
+metadata:
+  name: electrumx-snapshot
+spec:
+  volumeSnapshotClassName: electrumx
+  source:
+    persistentVolumeClaimName: electrumx-data-electrumx-0
```

Comment:
Same question about generating snapshots as for [bitcoind](https://github.com/keep-network/keep-core/pull/3573/files#r1223912755). Don't we need `-1` snapshots as well?

Reply:
Answered [here](https://github.com/keep-network/keep-core/pull/3573#discussion_r1224417079).